src/views/Profile/PersonalInfo/styles/PersonalInfo.styles.ts

```diff
@@ -0,0 +1,41 @@
+import { makeStyles, Theme } from '@material-ui/core';
+
+export const useStyles = makeStyles(
+  
+    (theme: Theme) => ({
+      PersonalInfo: {
+        // marginTop: theme.spacing(2),
+          '& .MuiFormControl-root' : {
+            marginTop: theme.spacing(1),
+            marginBottom: theme.spacing(1),
+          },
+        '& .MuiFormLabel-root':{
+          fontFamily: 'Raleway',
+          color: 'black',
+          fontWeight: 600,
+          fontSize: theme.spacing(3.6),
+          // lineHeight: theme.spacing(4.4),
```

Comment:
I think it's better to remove styles which are commented by now. They does not look that complex to keep them "just in case". And while they are commented - they are a tech debt 🤷‍♂️